composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -76,13 +90,12 @@ class KeywordInput extends React.Component {
 	/**
 	 * Handles changes in the KeywordInput.
 	 *
-	 * @param {Event} event The onChange event.
+	 * @param {SyntheticEvent} event The onChange event.
 	 *
-	 * @returns {void} Calls the checkKeywordInput-function.
+	 * @returns {void} Sets the state if a change has been made.
```

Comment:
Docblocks after a `@return {void}` isn't something I've seen before. Perhaps move this comment to the top of the docblock instead?